uiowa/uiowa
# Update sitenow_events image styles and single event page.

Resolves #2170 

### Testing
- Add an events block to a v3 site.
- Inspect the image and see it is responsive using the large image format.
- Go to a single event page and see the image is responsive using the original image.
- Provide feedback on different single event styles/layout.
- Double check HR.
